deployment/CP04OSPM_Deploy.csv

```diff
@@ -1,84 +1,93 @@
+AR-31,,AR-34,,CP04OSPM-RII01-02-ADCPSL010,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,,CGINS-ADCPSL-21393,39.93658333,-70.87933333,,408,453,
+AR-31,,AR-34,,CP04OSPM-SBS01-00-STCENG000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,,CGCON-BSTC01-00004,39.93658333,-70.87933333,,0,453,
+AR-31,,AR-34,,CP04OSPM-SBS11-02-MOPAK0000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,,CGINS-MOPAK0-11347,39.93658333,-70.87933333,,0,453,
+AR-31,,AR-34,,CP04OSPM-WFP01-00-WFPENG000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,39.93658333,-70.87933333,,407,453,
+AR-31,,AR-34,,CP04OSPM-WFP01-01-VEL3DK000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,39.93658333,-70.87933333,,407,453,
+AR-31,,AR-34,,CP04OSPM-WFP01-02-DOFSTK000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,39.93658333,-70.87933333,,407,453,
+AR-31,,AR-34,,CP04OSPM-WFP01-03-CTDPFK000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,39.93658333,-70.87933333,,407,453,
+AR-31,,AR-34,,CP04OSPM-WFP01-04-FLORTK000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,39.93658333,-70.87933333,,407,453,
+AR-31,,AR-34,,CP04OSPM-WFP01-05-PARADK000,10,1,2018-11-06T14:36:00,2019-04-22T16:13:00,CGMCP-04OSPM-00010,CGVEH-CWFPCP-99105,CGINS-PARADK-20513,39.93658333,-70.87933333,,407,453,
+AR-34,,,,CP04OSPM-RII01-02-ADCPSL010,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,,CGINS-ADCPSL-21446,39.9435,-70.8862,,415,435,
+AR-34,,,,CP04OSPM-SBS01-00-STCENG000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,,CGCON-BSTC01-00004,39.9435,-70.8862,,0,435,
+AR-34,,,,CP04OSPM-SBS11-02-MOPAK0000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,,CGINS-MOPAK0-99999,39.9435,-70.8862,,0,435,sn not on config
+AR-34,,,,CP04OSPM-WFP01-00-WFPENG000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.9435,-70.8862,,416,435,
+AR-34,,,,CP04OSPM-WFP01-01-VEL3DK000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.9435,-70.8862,,416,435,
+AR-34,,,,CP04OSPM-WFP01-02-DOFSTK000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.9435,-70.8862,,416,435,
+AR-34,,,,CP04OSPM-WFP01-03-CTDPFK000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.9435,-70.8862,,416,435,
+AR-34,,,,CP04OSPM-WFP01-04-FLORTK000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.9435,-70.8862,,416,435,
+AR-34,,,,CP04OSPM-WFP01-05-PARADK000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.9435,-70.8862,,416,435,
```

Comment:
Reviewed by CDobs. Looks good. Still need to verify instrument SNs.

Reply:
Have reviewed the instruments as well. All good.

Reply:
Andrew reviewed - OK